compiler/semcall.nim

```diff
@@ -354,7 +354,8 @@ proc getMsgDiagnostic(c: PContext, flags: TExprFlags, n, f: PNode): string =
 proc resolveOverloads(c: PContext, n, orig: PNode,
                       filter: TSymKinds, flags: TExprFlags,
                       errors: var CandidateErrors,
-                      errorsEnabled: bool): TCandidate =
+                      errorsEnabled: bool,
+                      diagnosticsFlag = true): TCandidate =
```

Comment:
Instead add yet another flag to `TExprFlags` or maybe there is one for this already.

Reply:
`efExplain` ? it use for `errorsEnabled` in `semOverloadedCall`

Reply:
I introduce `efNoDiagnostics` for this.